tests/test_bgp_neighbor_af.rb

```diff
@@ -407,7 +555,7 @@ def max_prefix(af, dbg)
     af.max_prefix_set(limit, threshold, warning)
     assert_equal(limit, af.max_prefix_limit,
                  "Test 4a. #{dbg} Failed to set limit to '#{limit}'")
-    assert_equal(threshold, af.max_prefix_threshold,
+    assert_equal((platform == :nexus ? threshold : 75), af.max_prefix_threshold,
```

Comment:
There is and it works, but this is testing threshold = nil. On XR it appears that the default is 75 though. So this is testing that.

max_prefix_threshold:
  cli_ios_xr:
    default_value: 75
  cli_nexus:
    default_value: ~